myhpi/static/scss/myHPI.scss

```diff
@@ -76,6 +76,16 @@ p {
     text-align: justify;
 }
 
+body {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}

```

Comment:
Please use the Bootstrap helper classes :)
https://getbootstrap.com/docs/5.0/utilities/flex/